web/src/stream_create_subscribers.ts

```diff
@@ -137,6 +136,7 @@ export function build_widgets(): void {
             return $(`#${CSS.escape("user_checkbox_" + user.user_id)}`);
         },
     });
+    pill_widget.appendValue(current_user.email);
```

Comment:
Hmm, why is it an email address we're using for the encoding here? User IDs are always better when both are available.